eclair-front/src/main/resources/application.conf

```diff
@@ -3,12 +3,15 @@ eclair {
 
   front {
     // To be overriden with the same key as the backend, so that the front has the same nodeid
-    priv-key-provider = "aws-sm" // aws-sm (AWS Secrets Manager) or env (environment variable)
+    priv-key-provider = "seed" // aws-sm (AWS Secrets Manager) or env (environment variable) or seed (seed.dat)
     priv-key = ${?NODE_PRIV_KEY} // used if priv-key-provider = env
     aws-sm.priv-key-name = "node-priv-key" // used if priv-key-provider = aws-sm
     // As a security measure, we also require the pub key, which will be matched against the priv key to make sure the
     // front is really using the expected key
     pub = ${NODE_PUB_KEY}
+
+    backend.ip = 127.0.0.1
+    backend.ip = ${?BACKEND_IP}
```

Comment:
That looks like a duplicate, the second one is probably the correct one? Or is it a way to set it from the environment variable if available, otherwise default to `127.0.0.1`?

Reply:
Yes that's a way to have a default (see https://github.com/lightbend/config/blob/master/HOCON.md#substitutions).